### PR TITLE
Meeting minutes, coordination meeting 201, 2021-02-04

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2021/2021-02-04-coordination.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+Should we just refer to these minutes or prepare another page as a 2021 *Plan of Work*?
+
+- Graeme will prepare a separate page and send the PR for people to comment on. This allows us to refer to topics from multiple meetings. Make it clear that this was a plan in early 2021, not set in stone!
+
+### Google Summer of Code
+
+Remember to [submit your project ideas](https://hepsoftwarefoundation.org/activities/gsoc.html) by 15 February. Avoid submitting the last minute, there is a review process for the PRs! Also check the instructions for changes to the projects this year. Few points to keep in mind:
+
+- The program [timeline](https://developers.google.com/open-source/gsoc/timeline) has changed. Student projects have halved in length (175 hours), so project proposals need to be more concise. The coding period is Jun 7 - Aug 16, but the project hours can be spread within as convenient for the mentor and student.
+- Shorter projects makes it possible to have more proposals splitting larger developments. This is encouraged as long as sub-projects do not depend on each other. There will be also potentially more candidates than last year.
```
Comment (amete) [inline organization/_posts/2021/2021-02-04-coordination.md:27]:
```suggestion
- Shorter projects makes it possible to have more proposals splitting larger developments. This is encouraged as long as sub-projects do not depend on each other. There will also be potentially more candidates than last year.
```

File: organization/_posts/2021/2021-02-04-coordination.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    * Thinking to invite reps from e.g. neutrinos, nuclear physics to describe what issues they are grappling with
+        * Graeme - Caterina would be a good contact with dark matter experiments
+* Reduced data formats (NanoAOD/DAOD_PHYSLITE)
+    * Attempted conversations with the non-mainstream analyses
+    * Should we revisit questions of policy (i.e. how to engage user base), consequences (framework design, sites)
+    * Also haven’t spoken to WLCG/DOMA in a year (<https://indico.cern.ch/event/890991/manage/timetable/#20200323>), try to reconnect? But needed more concreteness on what studies should be done.
+
+Besides these, we discussed commonalities with Training WG. One Q: how much should HSF branch into (experiment-agnostic) analysis tutorials?
+
+* Curriculum has a basic CMS example - without going heavily in on teaching physics, could this be expanded to cover statistical tools, demonstrate analysis ecosystem e.g. PyHEP? But don’t want to be prescriptive.
```
Comment (amete) [inline organization/_posts/2021/2021-02-04-coordination.md:63]:
```suggestion
* Curriculum has a basic CMS example - without focusing heavily on the physics aspects, could this be expanded to cover statistical tools, demonstrate analysis ecosystem e.g. PyHEP? But don’t want to be prescriptive.
```

File: organization/_posts/2021/2021-02-04-coordination.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    * Feeling that apart from experiment-specific SW, things like stats, generators etc. could be made more common between experiments
+    * No mention of HSF material now - probably because it is rather new compared to the established experimental tutorials?
+        * There is surely enough material on useful topics for beginners now that we should start to campaign for this as a useful core training resource
+
+### Detector Simulation
+
+- First new conveners' meeting last week
+- Initial set of topics to cover discussed
+    - Simulation challenges for future experiments (provisionally last week of Feb)
+    - NEST package for liquid nobel gases/dark matter
```
Comment (amete) [inline organization/_posts/2021/2021-02-04-coordination.md:74]:
```suggestion
    - NEST package for liquid noble gases/dark matter
```

File: organization/_posts/2021/2021-02-04-coordination.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+- Excellent participation - 94 people!
+- Tutorials being recorded in [dedicated HSF playlist](https://www.youtube.com/playlist?list=PLKZ9c4ONm-VnFUD0XX2DmfP1JA8VIRhXP).
+
+Next topical meeting on March 3rd (every first Wednesday of the month).
+
+- Happy to engage more strongly with the training WG for these tutorial sessions, which are rather training oriented.
+
+Not clear how many people followed-along live, launching and using the notebook at the same time as the speaker?
+
+- Graeme and Eduardo felt it was hard to listen to Jim, follow Slido and work with ones own notebook at the same time
```
Comment (amete) [inline organization/_posts/2021/2021-02-04-coordination.md:102]:
```suggestion
- Graeme and Eduardo felt it was hard to listen to Jim, follow Slido and work with one's own notebook at the same time
```

File: organization/_posts/2021/2021-02-04-coordination.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+1. GitHub Actions CI/CD training 16-19th Feb. Mentors and participants still welcome! <https://indico.cern.ch/event/1001128>
+2. Please drop into our weekly training meeting 16h CET Mondays to kick-start a training activity.
+
+
+### Event Generators
+
+- Have started a discussion with Markus Dienfenthaler on possible needs of NP community.
+- We understood that we can't get reliable HS06 values from CMS monitoring - after testing for the first time a single big GEN campaign that has the full info (i.e. not affected by the 18 month limit) - an issue that has been rolling on since Nov 2018 WS....
+    - Instead, CMS is now going through a campaign with local runs where CPU->HS06 conversion is known and everything is under control. After some checks, we will discuss in an HSF GEN meeting.
+- Did not make it to the Training WG meeting this week. Ideally need to find time to discuss amongst ourselves first, but hope to discuss Generators needs in that meeting soon.
```
Comment (amete) [inline organization/_posts/2021/2021-02-04-coordination.md:131]:
```suggestion
- Did not make it to the Training WG meeting this week. Ideally we need to find time to discuss amongst ourselves first, but hope to discuss Generators needs in that meeting soon.
```

File: organization/_posts/2021/2021-02-04-coordination.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+- In response to paper referee comment have been significantly expanding the discussion on MP/MT usage in experiments for generators.
```
Comment (amete) [inline organization/_posts/2021/2021-02-04-coordination.md:132]:
```suggestion
- In response to paper referee comment, we have been significantly expanding the discussion on the MP/MT usage in experiments for generators.
```

File: organization/_posts/2021/2021-02-04-coordination.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+- Les Houches 2021 (<https://phystev.cnrs.fr>) will include some projects on generator software, including benchmarking, ML applications and migration to alterative architectures (largely GPU).
```
Comment (amete) [inline organization/_posts/2021/2021-02-04-coordination.md:133]:
```suggestion
- Les Houches 2021 (<https://phystev.cnrs.fr>) will include some projects on generator software, including benchmarking, ML applications and migration to alternative architectures (largely GPU).
```

File: organization/_posts/2021/2021-02-04-coordination.md [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+
+---
+
+## AOB
+
+### Meetings next week:
+
+- Software and Computing Roundtable, 17h Tuesday 9 Feb, <https://indico.jlab.org/event/420/>
+    - I/O is the topic
+- Compute Accelerator Forun, 16h30 Wednesday 10 Feb, <https://indico.cern.ch/event/975007/>
```
Comment (amete) [inline organization/_posts/2021/2021-02-04-coordination.md:163]:
```suggestion
- Compute Accelerator Forum, 16h30 Wednesday 10 Feb, <https://indico.cern.ch/event/975007/>
```

**Review by amete — APPROVED** [review]
I have a few minor suggestions but apart from those it looks good to me. Thanks a lot @graeme-a-stewart.

**Comment by graeme-a-stewart** [comment]
Thanks for the careful reading, @amete

**Pull request merged**